source/docs/software/advanced-controls/controls-glossary.rst

```diff
@@ -15,15 +24,36 @@ Controls Glossary
       Used in position or negative feedback with a :term:`plant` to bring about a desired :term:`system state <state>` by driving the difference between a :term:`reference` signal and the :term:`output` to zero.
 
+   convolution
+      A mathematical operation that calculates a weighted moving average of one function, with the weights assigned by a second function. A common way to "filter" sensor input is to apply a *convolution* to it, using a carefully-chosen filtering function. See `convolution <https://en.wikipedia.org/wiki/Convolution>`__. on wikipedia for more info.
+
+   counter-electromotive force
+      A :term:`voltage` generated in a spinning motor. The voltage is a result of the fact that has a coil of wire rotating near a magnet. See `Counter-electromotive_force <https://en.wikipedia.org/wiki/Counter-electromotive_force>`__ on wikipedia for more info.
+
+   current
+      The flow of electrons through a conductor. Current is described with a unit of "Amps" (or simply "A"), and is measured at a single point in a circuit. One amp is equal to :math:`6241509074000000000` electrons moving past the measurement point in one second.
+
    dynamics
       A branch of physics concerned with the motion of bodies under the action of forces. In modern control, systems evolve according to their dynamics.
 
+   derivative
+      A mathematical operation which evaluates the "rate-of-change" of a function at a given point. See `derivative <https://en.wikipedia.org/wiki/Derivative>`__ on wikipedia for more info.
+
    error
       :term:`Reference <reference>` minus an :term:`output` or :term:`state`.
 
+   exponential search
+      An iterative process of finding a specific value within a wide search range by applying a multiplicative factor to the search value. See `exponential search <https://en.wikipedia.org/wiki/Exponential_search>`__ on wikipedia for more info.
+
+   exponential smoothing
+      The usage of exponential window function in a convolution with an input signal. It is a very common way to implement a simple low-pass filter. See `exponential smoothing <https://en.wikipedia.org/wiki/Exponential_smoothing>`__ on wikipedia for more info.
+
    gain
       A proportional (scalar) value that relates the magnitude of an input signal to the magnitude of an output signal. In the signal-dimensional case, gain can be thought of as the proportional term of a PID controller. A gain greater than one would amplify an input signal, while a gain less than one would dampen an input signal. A negative gain would negate the input signal.
 
+   Gaussian distribution
+      A special mathematical function useful for statistical analysis. The graph of a Gaussian function is a "bell curve" shape. This function is described by its mean (the location of the "peak" of the bell curve) and variance (a measure of how "spread out" the bell curve is). See `Gaussian distribution <https://en.wikipedia.org/wiki/Gaussian_function>`__  on wikipedia for more info.
```

Comment:
Expand first sentence to: "A special mathematical function that describes distributions of *average* values.  As more and more data goes into an average, its distribution will almost always get closer to a normal distribution, even if the individual components of the average do not themselves follow a normal distribution."

I think it's important to mention in a heuristic way *why* the Gaussian is "special."  There are a lot of reasons, but this one's kind of the easiest to grok without a lot of extra math background.